lib/rubocop/cop/bundler/duplicated_group.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bundler
+      # A Gem group should be listed only once in a Gemfile.
+      # @example
+      #   # bad
+      #   group :development do
+      #     gem 'rubocop'
+      #   end
+      #
+      #   group :development do
+      #     gem 'rubocop-rails'
+      #   end
+      #
+      #   # bad
```

Comment:
It'd be nice to explain why this is bad and the next example is good, as it's not obvious to me.

Reply:
(to be clear - it took me a moment to see that the names were the same, but in different order)